test/data/config-multi-line

```diff
@@ -0,0 +1,20 @@
+macro p open;\
+reload;\
+quit;\
+quit -- "opens, reloads, quits, then quits newsboat"
+
+macro j qui\
+t;\
+qu\
+it;\
+q\
+uit;\
+quit -- "Dummy macro"
+
+
+macro f open;  \
+reload;        \
+quit;          \
+quit  -- "Spaces before \ are ok. Not after, if concatenation is what the user is after. Otherwise treated as-is"
+
+macro a open; quit; reload -- "some macro description"
```

Comment:
Please modify the test to check those as well, or add a comment there why these aren't tested, or drop them altogether.

Reply:
I've removed the redundant data in the config file